# sudo diff -r locklite locklite-prod -x .idea -x node_modules -x .git -x prod.diff > locklite/prod.diff
Seulement dans locklite-prod: .env
Seulement dans locklite: .env.prod
Seulement dans locklite-prod: .next
Seulement dans locklite-prod: next-env.d.ts
diff -r -x .idea -x node_modules -x .git -x prod.diff locklite/src/app/ui/login/page.tsx locklite-prod/src/app/ui/login/page.tsx
8c8
< import { Alert, Typography } from '@mui/material';
---
> // import { Alert, Typography } from '@mui/material';
13,19c13,19
<       <Alert variant="filled" severity="error">
<         This application is{' '}
<         <Typography component="span" fontWeight="bold">
<           fictitious
<         </Typography>
<         . We strongly recommend that you do not enter your real login details.
<       </Alert>
---
>       {/*<Alert variant="filled" severity="error">*/}
>       {/*  This application is{' '}*/}
>       {/*  <Typography component="span" fontWeight="bold">*/}
>       {/*    fictitious*/}
>       {/*  </Typography>*/}
>       {/*  . We strongly recommend that you do not enter your real login details.*/}
>       {/*</Alert>*/}
Seulement dans locklite-prod/src: middleware.dev.ts
